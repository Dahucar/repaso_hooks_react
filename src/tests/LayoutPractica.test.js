import React from 'react';
const { shallow } = require("enzyme");
const { LayoutPractica } = require("../components/05-useLayoutEffect/LayoutPractica");

describe('Test in 05-hooks components ', () => {
    const wrapper = shallow( <LayoutPractica  /> );     
    test('Should be macht with the Snapshot', () => {
        expect( wrapper ).toMatchSnapshot();
    });
});
