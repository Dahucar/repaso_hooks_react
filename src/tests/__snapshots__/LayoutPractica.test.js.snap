// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in 05-hooks components  Should be macht with the Snapshot 1`] = `
<div
  className="container card"
>
  <div
    className="card-header"
  >
    <h2>
      LayoutPractica
    </h2>
  </div>
  <div
    className="card-body"
  >
    <div>
      <blockquote
        className="blockquote text-right"
      >
        <p
          className="mb-0"
        >
           quote - 
          1
           
        </p>
        <p
          className="mb-0"
        >
           
           
        </p>
      </blockquote>
      <pre>
        {}
      </pre>
      <button
        className="btn btn-success"
        onClick={[Function]}
      >
        Siguiente
      </button>
      <button
        className="btn btn-danger"
        onClick={[Function]}
      >
        Volver
      </button>
    </div>
  </div>
</div>
`;
